src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CrlCache.cs

```diff
@@ -83,9 +85,14 @@ private static bool AddCachedCrl(X509Certificate2 cert, SafeX509StoreHandle stor
                         return false;
                     }
 
-                    // TODO (#3063): Check the return value of X509_STORE_add_crl, and throw on any error other
-                    // than X509_R_CERT_ALREADY_IN_HASH_TABLE
-                    Interop.Crypto.X509StoreAddCrl(store, crl);
+                    if (!Interop.Crypto.X509StoreAddCrl(store, crl))
+                    {
+                        // Ignore cert already store throw on anything else, in both cases the error queue will be cleared.
```

Comment:
Nit: I'm having trouble parsing this sentence. "Ignore cert already stored and throw on anything else"?